### PR TITLE
raylib: Update to 5.5

### PR DESCRIPTION
**Pull request by lxmcf**
* Have updated bindings to match the newly released raylib 5.5
* Removed separate libraries for MacOS x86 and MacOS ARM64 as official raylib binaries are universal binaries
* Removed 'IsMouseButtonUp' workaround

### CODE REVIEW

File: vendor/raylib/raylib.odin [hunk 1]
```diff
@@ -726,6 +732,9 @@ ShaderLocationIndex :: enum c.int {
 	MAP_IRRADIANCE,                   // Shader location: samplerCube texture: irradiance
 	MAP_PREFILTER,                    // Shader location: samplerCube texture: prefilter
 	MAP_BRDF,                         // Shader location: sampler2d texture: brdf
+    VERTEX_BONEIDS,                   // Shader location: vertex attribute: boneIds
```
Comment (Kelimion) [inline vendor/raylib/raylib.odin:735]: More indentation issues.

**Comment by Kelimion** [comment]
Looks good aside from those couple of indentation issues.

**Comment by lxmcf** [comment]
These should now be resolved, did another once over and could not see any further indentation issues.

**Comment by Kelimion** [comment]
I'll go make coffee and wait for the CI, then merge. Thanks.

**Pull request merged**